themes/dark-and-light/v1/theme.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetwriters/notesnook-themes/main/schemas/v1.schema.json",
+  "name": "DARK & light",
```

Comment:
Is this casing intended?

Reply:
Yes. At least till the time, I find a better name for it😁